backend/src/main/resources/api-definition.json

```diff
@@ -27,6 +27,16 @@
         "produces": [
           "*/*"
         ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "Authorization",
+            "description": "JWT to authenticate the user, preceded with the string 'Bearer '",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
         "responses": {
           "200": {
```

Comment:
Now it should also be possible to get 403s, right?

Reply:
Yes, it's already in there ;)